lib/iris/_concatenate.py

```diff
@@ -357,6 +358,31 @@ def __init__(self, cube):
             else:
                 self.scalar_coords.append(coord)
 
+    def _cubes_equal(self, other):
+        """
+        Determine whether the two *cubes* are identical by comparing their
```

Comment:
I don't think we should use the word "identical".
Issues of identity/equality/equivalence are always a bit confused in Python,
but I think "identical" should be reserved to mean "A **is** B".